crypto/hmac/hm_meth.c

```diff
@@ -152,13 +152,16 @@ static int hmac_ctrl_str(EVP_MAC_IMPL *hctx, const char *type,
 {
     if (!value)
         return 0;
+#ifndef FIPS_MODE
+    /* Not supported in FIPS mode due to the implict fetch of the md */
     if (strcmp(type, "digest") == 0) {
         const EVP_MD *d = EVP_get_digestbyname(value);
 
         if (d == NULL)
             return 0;
         return hmac_ctrl_int(hctx, EVP_MAC_CTRL_SET_MD, d);
     }
+#endif
```

Comment:
This is problematic, 'cause directly fetched MDs are broken for the moment, since sizes and flags haven't been transferred.  I need to get back to the fixing PR, thanks for the reminder...

Reply:
Has this request been resolved?

Reply:
I think this isn't so much a request to do anything on this PR...rather a side note.